x-pack/plugins/reporting/common/constants.ts

```diff
@@ -113,3 +113,5 @@ export enum JOB_STATUSES {
   CANCELLED = 'cancelled',
   WARNINGS = 'completed_with_warnings',
 }
+
+export const UNVERSIONED_VERSION = '7.14.0';
```

Comment:
I think for future reference it'd be nice to have a verbose comment above this value explaining why it exists and what it means :)